packages/tooltip/src/main.js

```diff
@@ -118,15 +123,26 @@ export default {
       this.timeout = setTimeout(() => {
         this.showPopper = true;
       }, this.openDelay);
+
+      if (this.hideAfter > 0) {
+        this.timeoutPending = setTimeout(() => {
+          this.showPopper = false;
+        }, this.hideAfter);
+      }
     },
 
     handleClosePopper() {
       if (this.enterable && this.expectedState || this.manual) return;
-      clearTimeout(this.timeout);
```

Comment:
I think `clearTimeout(this.timeout);` should not be removed.

Reply:
You don't want to remove the timeout when the tooltip is closed?  Why not?

Reply:
I want to remove the timeout when the tooltip is closed. This is exactly what this line does, so it should not be removed.

Reply:
ok... was the comment directed to me?  I'm confused.

Reply:
I see what you mean.  I removed that line.  The line is restored.